controllers/hostedcontrolplane/hostedcontrolplane.go

```diff
@@ -62,6 +63,9 @@ const (
 	dynatraceSecretName      = "dynatrace-token"
 	dynatraceApiKey          = "apiToken"
 	dynatraceTenantKey       = "apiUrl"
+
+	// requeueAfter can be used with RequeueAfter()
+	requeueAfterValue = 5 * time.Minute
```

Comment:
Would something like `vpcEndpointRetryTimeout` maybe be a more descriptive variable value here?

Reply:
Thanks! I like that a lot better (naming is hard) - change made